src/Test/ProxyTestCase.php

```diff
@@ -11,190 +11,15 @@
 
 namespace FOS\HttpCache\Test;
 
-use FOS\HttpCache\Test\PHPUnit\IsCacheHitConstraint;
-use FOS\HttpCache\Test\PHPUnit\IsCacheMissConstraint;
-use Http\Adapter\HttpAdapter;
-use Http\Discovery\HttpAdapterDiscovery;
-use Http\Discovery\MessageFactoryDiscovery;
-use Http\Discovery\UriFactoryDiscovery;
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\UriInterface;
 
 /**
- * Abstract caching proxy test case
- *
+ * Abstract test that provides an application HTTP client and cache assertions
  */
 abstract class ProxyTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
should we drop this class? for just the two use trait statements it seems a bit overcomplicated.